### ISSUE TITLE
try to fix build

### ISSUE THREAD

**Issue by ltalirz** [marvel-nccr/ansible-role-aiida#1]
build of this role [is broken](https://travis-ci.org/marvel-nccr/ansible-role-aiida/jobs/442311285#L541) (on docker only) already since a long time due to a

```
Failed to set permissions on the temporary files 
Ansible needs to create when becoming an unprivileged user 
(rc: 1, err: chown: changing ownership of '/var/tmp/ansible-tmp-1539713856.1-211737224251679/': Operation not permitted
chown: changing ownership of '/var/tmp/ansible-tmp-1539713856.1-211737224251679/AnsiballZ_postgresql_db.py': Operation not permitted
}). 
For information on working around this, see https://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user
```

One way around this is supposedly to have a file system with acls and the presence of the acl tool.
We are currently trying to do this by

https://github.com/marvel-nccr/ansible-role-aiida/blob/master/.travis.yml#L31-L33

but it doesn't seem to work.

**Comment by ltalirz** [comment]
the switch to molecule has fixed this problem.
all that is left to do is fix idempotency

**Comment by ltalirz** [comment]
fixed in 0972c67e34891d9eff68b46cdef7d3ae5fefb186
